### ISSUE TITLE
Handling "deviceready" event with "cordova" service

### ISSUE THREAD

**Issue by adamniedzielski** [isleofcode/ember-cordova#59]
Thank you for creating this addon!

I would like to report one inconsistency in how `deviceready` event is handled. In Cordova docs (https://cordova.apache.org/docs/en/latest/cordova/events/events.html#deviceready):

> The `deviceready` event behaves somewhat differently from others. Any event handler registered after the `deviceready` event fires has its callback function called immediately.

However, from what I have observed, this is not the case when using `cordova` service:

```javascript
// this invokes "vibrate" function
document.addEventListener("deviceready", this.vibrate.bind(this), false);
// this does not invoke "vibrate" function presumably because the event already fired long time ago
this.get("cordova").on("deviceready", this, "vibrate");
```

What do you think? Should we match the "unwrapped" behaviour?

**Comment by runspired** [comment]
We should match, I've been intending to fix. I believe one of these two fixes was applied and the other forgotten: 

Fix #1: https://github.com/isleofcode/ember-cordova/pull/39#issuecomment-219839761
Fix #2: https://github.com/isleofcode/ember-cordova/pull/39#issuecomment-219945388

**Comment by AlexBlom** [comment]
Thanks, this is a fair issue. I'll put it into this dev sprint, now that I've spiked an icon branch.
